lib/src/git.rs

```diff
@@ -208,13 +217,18 @@ pub fn import_some_refs(
     let changed_git_refs = diff_refs_to_import(mut_repo.view(), git_repo, git_ref_filter)?;
 
     // Import new heads
-    for id in changed_git_refs
-        .values()
-        .flat_map(|(_, new_git_target)| new_git_target.added_ids())
-    {
-        prevent_gc(git_repo, id)?;
-        let commit = store.get_commit(id).unwrap();
-        mut_repo.add_head(&commit);
+    for (ref_name, (_, new_git_target)) in &changed_git_refs {
+        for id in new_git_target.added_ids() {
+            prevent_gc(git_repo, id)?;
+            let commit =
+                store
+                    .get_commit(id)
+                    .map_err(|err| GitImportError::MissingRefAncestor {
+                        ref_name: ref_name.to_string(),
+                        err,
+                    })?;
+            mut_repo.add_head(&commit);
```

Comment:
Better to `get_commit(id)` first? I think `prevent_gc()` would also fail if `id` was missing.

Reply:
Ah, that's better. Thanks!